03 Writing Algorithms/03 Securities/01 Key Concepts/06 Symbol Properties.html

```diff
@@ -1,10 +1,14 @@
 <p>The <code class="csharp">SymbolProperties</code><code class="python">symbol_properties</code> property of a <code>Security</code> contains properties for that specific security, such as the lot size and the minimum price variation.</p>
 
 <div class="section-example-container">
-    <pre class="csharp">var symbolProperties = Securities[_symbol].SymbolProperties;
+    <pre class="csharp"> // Access the lot size and the minimum price variation through the symbol_properties
```

Comment:
- Full stops
 - Try and include WHY something is useful in the comment not just what its doing.

e.g. 
" Round orders to a brokers lot size and minimum price variation using the symbol_properties."